metricbeat-tests/README.md

```diff
@@ -0,0 +1,214 @@
+# Functional tests for Metricbeat Integrations
```

Comment:
I assume we are focusing on metricbeat at this point, but the scope woul also include filebeat and in the future the new "integrations" (which will involve greater kibana interaction to install them)

Reply:
Yes. As this POC is for metricbeat, we are referring just to it everywhere. We can eventually change to cover all beats family